src/drivers/localstorage.js

```diff
@@ -123,7 +124,7 @@
                     value = serializer.deserialize(value);
                 }
 
-                value = iterator(value, key.substring(keyPrefixLength), i + 1);
+                value = iterator(value, key.substring(keyPrefixLength), iterationNumber++);
```

Comment:
Ahhhh, okay. You're right, I was being dense.

Oddly the test doesn't actually test for it, so I'll rework it and merge it in afterward. Thanks :smiley: